prototypes/entity.lua

```diff
@@ -19,6 +19,43 @@ if settings.startup["Warehousing-sixteen-mode"].value then
 	storage_storehouse_slots = 300
 end
 
+function shiftX(modifier_x, x)
+	return modifier_x + x
+end
+function shiftY(modifier_y, y)
+	return modifier_y + y
+end
```

Comment:
These functions are redundant. They take the same arguments and return the same result, just with different names.

Honestly, it would be clearer to just embed the arithmetic below, instead of obfuscating it with these. I think it would also be clearer (and cleaner) to put these shift vectors in the declared prototypes.

Reply:
I realized these were redundant later on but I ran out of time to continue working on it.

The reason I wanted the shift vectors outside calling a function was so if we needed to move it all we had to do was change 2 values instead of each individual sprite. So it was an attempt to future proof the new models using these.

Reply:
I'm slow lol, I understand what you mean and it will still work the same as I have it now haha. I still want to screw around with it to see if there's any way I can make this cleaner with a function to generate the Sprite callbacks. Or possibly put it in a new file and call that file into the entity.lua file. Not sure yet what will be easiest and cleanest to function as efficiently as possible. Another puzzle for me to solve 😊😊